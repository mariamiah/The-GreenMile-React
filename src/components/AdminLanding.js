/* eslint-disable react/prop-types */
/* eslint-disable react/destructuring-assignment */
import React from 'react';
import parseDate from '../utils/helpers';

const AdminLanding = props => (
<div className="container">
  <div className="row">
    <div className="col-sm-6 bg-success">
      <p>Lorem ipsum...</p>
    </div>
    <div className="col-sm-6 bg-warning">
      <p>Sed ut perspiciatis...</p>
    </div>
  </div>
</div>
);
export default AdminLanding;




