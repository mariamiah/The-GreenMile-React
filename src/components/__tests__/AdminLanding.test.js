/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import AdminLanding from '../AdminLanding';

describe('<AdminLanding />', () => {
  const props = {
    item: {
      category_name: 'Denim',
      date_added: '12/01/2019',
      price: '10000',
      product_code: '123456',
      product_name: 'shorts',
      product_quantity: '10',
    },
  };
  it('should render without crashing', () => {
    const wrapper = shallow(<AdminLanding {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
