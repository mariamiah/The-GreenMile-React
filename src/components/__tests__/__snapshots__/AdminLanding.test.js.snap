// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminLanding /> should render without crashing 1`] = `ShallowWrapper {}`;
